### PR TITLE
fix: build error on flutter 3.24.0

### PR DESCRIPTION
**Pull request by ktakayama**
Fix the following error that occurs when compiling for an Android build on Flutter 3.24.0

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':external_app_launcher:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR: /Users/takayama/ghq/github.com/medpeer-dev/navel/build/external_app_launcher/intermediates/merged_res/release/values/values.xml:194: AAPT: error: resource android:attr/
lStar not found.
```

### CODE REVIEW

File: android/build.gradle
Comment (bigbenyayi) [inline android/build.gradle:29]: I don't think the `minSdkVersion` needs to be changed for this package to compile on Flutter 3.24.0. You can leave it at 16.

Comment (bigbenyayi) [reply android/build.gradle:29]: LGTM! Thanks for creating this PR

**Review by bigbenyayi — APPROVED** [review]

**Comment by kunatastic** [comment]
Hey team!!

Any update on this

**Comment by josephmangmang** [comment]
Hi also same issue

**Comment by bonellia** [comment]
Any plans to merge this soon? Thanks in advance.

**Comment by bonellia** [comment]
Hi @bigbenyayi 
It seems @ktakayama is inactive.
I see that you already approved these changes. Any chance you can merge it?

**Comment by bigbenyayi** [comment]
I sadly do not have the rights to merge on this project

**Comment by ktakayama** [comment]
I can't merge this PR because I don't have write access to this repository

**Comment by bonellia** [comment]
I see, thanks anyways. We ended up using URL launcher plugin for those who are interested in.
We can launch store page if said application is not installed. Here is a sample usage:
```Dart
canLaunchUrl(Uri.parse('spotify://')).then((bool result) {
  if (result == true) {
    launchUrl(Uri.parse('spotify://'),
        mode: LaunchMode.externalApplication);
  } else {
    if (Platform.isAndroid) {
      String appPackageName = 'com.spotify.music';
      launchUrl(Uri.parse("https://play.google.com/store/apps/details?id=" + appPackageName));
    }
    if (Platform.isIOS) {
      launchUrl(Uri.parse('itms-apps://itunes.apple.com/us/app/spotify-music-and-podcasts/id324684580'));
    }
  }
});
```

**Comment by geekruchika** [comment]
Will merge this soon.

**Review by geekruchika — APPROVED** [review]

**Comment by geekruchika** [comment]
Thanks for PR. Already added fix in 4.0.1 version.

**Pull request closed without merging**
